qa/README.md

```diff
@@ -5,6 +5,17 @@ Every pull request to the bitcoin repository is built and run through
 the regression test suite. You can also run all or only individual
 tests locally.
 
+Test dependencies
+=================
+Before running the tests, the following must be installed.
+
+Unix
+----
+The python-zmq library is required. On Ubuntu or Debian it can be installed via: 
+```
+sudo apt-get install python-zmq
+```
+
 Running tests
```

Comment:
Nit: As you are already touching this file, mind to add a line to mention an undocumented arg?

`--coverage` (this generates a basic coverage report for the RPC)